pom.xml

```diff
@@ -151,17 +151,25 @@
       </properties>
     </profile>
 
-    <profile>  <!-- compile error on com.google.common.cache in hive module, with -Phadoop-1.x -->
-      <id>hive-.10</id>
-      <properties>
-        <hive.version>0.10.0</hive.version>
-      </properties>
-    </profile>
     <profile>
-      <id>hive-.11</id>
+      <id>hive-.11</id>  <!-- Need to build with Hive-0.12 before testing with 0.11 -->
       <properties>
         <hive.version>0.11.0</hive.version>
       </properties>
+      <!-- NEEDED with Hive-0.11  dependencies>
+	<dependency>
+	  <groupId>javax.jdo</groupId>
+	  <artifactId>jdo2-api</artifactId>
+	  <version>2.3-ec</version>
+	</dependency>
+      </dependencies>
+      <repositories>
+	<repository>
+	  <id>org.datanucleus</id>
+	  <name>datanucleus</name>
+	  <url>http://www.datanucleus.org/downloads/maven2/</url>
+	</repository>
+      </repositories -->
     </profile>
```

Comment:
Building and testing with Hive-0.11 is barely supported any more - multiple manual steps needed.